torch/distributed/_sharded_tensor/ops/embedding_bag.py

```diff
@@ -331,12 +432,21 @@ def _handle_row_wise_sharding(
             if gathered_per_sample_weights is not None
             else None
         )
+        # If input is None, passing in max_norm causes
+        # errors in CUDA.
+        if max_norm is not None and inp.size(0) == 0:
+            max_norm = None
```

Comment:
What is the behavior for `nn.EmbeddingBag` when this happens? We should replicate that behavior here, silently setting `max_norm=None` doesn't seem right.

Reply:
Oh, this is fine because the input here is None. Interestingly, it throws exception if we don't set it as None. And for embedding bag, it does not raise exception. (It took me some time to learn this difference). So do you have any suggestion on how we should handle this?

Reply:
Ah so if input is empty and max_norm is not None, the embedding_bag lookup will throw an exception? Btw, if input is empty do we even need to do the embedding lookup?